neuraxle/hyperparams/distributions.py

```diff
@@ -390,6 +390,34 @@ def narrow_space_from_best_guess(self, best_guess, kept_space_ratio: float = 0.5
             return FixedHyperparameter(best_guess).was_narrowed_from(kept_space_ratio, self)
         return Uniform(new_min_included, new_max_included).was_narrowed_from(kept_space_ratio, self)
 
+    def pdf(self, x_value):
+        """
+        Calculate the probability distribution value at position x_value.
```

Comment:
Could we be more clear than "calculate value at position x_value"? (the double usage of the general word value makes me doubt as a reader, perhaps rename variable or change docstring).